bazel/extra_deps.bzl

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
@rochaudhari   As per the[ CNCF guidelines](https://github.com/cncf/foundation/blob/master/copyright-notices.md), the copyright header should be - " Copyright The OpenTelemetry Authors", as we have it all other files within this repo. These two lines for copyright and license should be enough:

```
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0
```